frontend/src/metabase/nav/components/SearchResults.tsx

```diff
@@ -44,19 +45,17 @@ export const SearchResults = ({
     [searchText],
   );
 
-  const query = useMemo(
-    () => ({
-      q: debouncedSearchText,
-      limit: DEFAULT_SEARCH_LIMIT,
-      ...searchFilters,
-      models: models ?? searchFilters.type,
-    }),
-    [debouncedSearchText, searchFilters, models],
-  );
+  const query = {
```

Comment:
The `useMemo` is unnecessary since the hook uses `useDeepCompareEffect` to check if `query` has changed